docs/source/user_guide/pipelines.md

```diff
@@ -66,10 +66,28 @@ The [tutorials](/getting_started/tutorials.md) provide comprehensive step-by-ste
 
    ![Pipeline editor links in launcher](../images/user_guide/pipelines/editor-links.png)
 
-1. Expand the properties panel and define the pipeline properties. Pipeline properties include a description and default values for certain node properties. When a default property is set, individual nodes can override the value of these properties in their node property tab, described below. (Support for pipeline properties varies by release.)
+#### Defining pipeline properties
+
+1. Expand the properties panel and select the 'pipeline properties' tab.
 
    ![Pipeline properties](../images/user_guide/pipelines/pipeline-properties.png)
 
+   Pipeline properties include:
+      - An optional description, summarizing the pipeline purpose. 
+      - Properties that apply to every generic pipeline node. In this release the following properties are supported:
+        - **Object storage path prefix**. Elyra stores pipeline input and output artifacts in a cloud object storage bucket. By default these artifacts are located in the `/<pipeline-instance-name>` path. The example below depicts the artifact location for several pipelines in the `pipeline-examples` bucket:
+          ![artifacts default storage layout on object storage](../images/user_guide/pipelines/node-artifacts-on-object-storage.png)
+
+          Configure an object storage path prefix to store artifacts in a pipeline-specific location `/<path-prefix>/<pipeline-instance-name>`:
+          ![artifacts custom storage layout on object storage](../images/user_guide/pipelines/generic-node-artifacts-custom-layout.png)
+
+      - Default values that apply to every generic pipeline node. These values can be overridden for each node.
+         - **Runtime image**. Identifies the container image used to execute the Jupyter notebook or script. Select an image from the list or [add a new one](runtime-image-conf.md) that meets your requirements.
+         - **Environment variables**. A list of environment variables to be set in the container that executes the Jupyter notebook or script. Format: `ENV_VAR_NAME=value`. Entries that are empty (`ENV_VAR_NAME=`) or malformed are ignored.
+         - **Data volume**. A list of [Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (PVC) to be mounted into the container that executes the Jupyter notebook or script. Format: `/mnt/path=existing-pvc-name`. Entries that are empty (`/mnt/path=`) or malformed are ignored. The referenced PVCs must exist in the Kubernetes namespace where the generic pipeline nodes are executed.
```

Comment:
Should we note that these are ignored when pipelines are run locally?

We could probably add a link for _generic pipeline nodes_ here as well for emphasis.